doc/usage/debugger/launch.json.template

```diff
@@ -0,0 +1,46 @@
+{
+    // MCAD Debugger configuration for VSCode
+
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch MCAD Controller",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "cmd/kar-controllers/main.go",
+            //"buildFlags": "-race",
+            "env": {
+                "DISPATCH_RESOURCE_RESERVATION_TIMEOUT":"120000",
+                "PREEMPTION":"false",
+                "DISPATCHER_MODE":"false",
+                "QUOTA_ENABLED":"false",
+            },
+            "args": ["--kubeconfig", "</path/to/your/kube/config>", "--v", "15", "--logtostderr"], 
```

Comment:
```suggestion
            // To set the desired log level set "--v", "<desired_logging_level>" in the args.
            "args": ["--kubeconfig", "</path/to/your/kube/config>", "--v", "15", "--logtostderr"], 
```
Maybe we can add a comment that explains how to set the log level?

Reply:
Great idea, thanks Mark!